Sources/WalletConnect/WalletConnectError.swift

```diff
@@ -1,149 +1,50 @@
-// 
-
-import Foundation
-
-// TODO: Migrate protocol errors to ReasonCode enum over time. Use WalletConnectError for client errors only.
 enum WalletConnectError: Error {
     
+    case pairingProposalFailed(Error)
+    case malformedPairingURI
+    case noPairingMatchingTopic(String)
     case noSessionMatchingTopic(String)
     case sessionNotSettled(String)
+    case invalidCAIP10Account(String)
```

Comment:
this could be removed then if we use Account type as mentioned above

Reply:
Error case removed